Tweets/_posts/What is Yearn/2021-10-09-What-is-Yearn.md

```diff
@@ -4,95 +4,112 @@ title:  "What is Yearn"
 categories: [ Tweets ]
 image: ./What-is-Yearn/3_1446243220468293632.jpg
 author: Wot_Is_Goin_On
-translator:
-publish: false
+translator: japzar
+publish: true
 ---
 
 original source: [twitter](https://twitter.com/Wot_Is_Goin_On/status/1446540007292952579)
 
-##### 1
-People had difficulty understanding ETH in mid-2020. It wasn’t clear what the use case would be, TVL was at $8bn and hadn’t grown much in 18 months. Even out of those who believed in Ethereum, there was scepticism around how value would accrue to ETH.
+## 1/n
 
-##### 2
-Fast forward to today and it’s clear there’s a lot that can be built on Ethereum (DeFi, NFTs and gaming), TVL has sky-rocketed to over $100bn and throw in a token burn mechanism and people understand how value accrues to the native token.
+La gente tenía dificultades para entender ETH a mediados de 2020. No estaba claro cuál sería su uso, su valor total estaba en $8MM y no había crecido mucho en 18 meses. Incluso entre aquellos que creían en Ethereum, había escepticismo sobre cómo se acumularía valor para ETH.
 
-##### 3
-In some ways the perception of Yearn today is similar to that of ETH in mid-2020, at least in terms of uncertainty over what will be built on Yearn, its future growth and value accrual to YFI. Even Yearn’s OGs are getting restless.
+## 2/n
+
+Hoy en día está claro que hay mucho que se puede desarrollar en Ethereum (Defi, NFTs y juegos), el valor total (TVL) se ha disparado a más de $100MM y ha lanzado un mecanismo de quema de tokens y la gente comprende cómo se acumula el valor al token nativo.
+
+## 3/n
+
+De laguna manera, la percepción de Yearn hoy es similar a la de ETH a mediados de 2020, al menos en términos de incertidumbre sobre lo que se construirá en Yearn, su crecimiento futuro y la acumulación de valor para YFI. Incluso los fanáticos más viejos de Yearn están inquietos. [pic.twitter.com/TQDFz74CFc](https://twitter.com/Wot_Is_Goin_On/status/1446540013576065026/photo/1)
```

Comment:
AS-IS: "De laguna manera"
TO-BE: "De alguna manera"

Reply:
fixed